Gems/ScriptCanvas/Code/Editor/View/Widgets/NodePalette/NodePaletteModel.cpp

```diff
@@ -35,6 +35,12 @@ AZ_DEFINE_BUDGET(NodePaletteModel);
 
 namespace
 {
+    static constexpr char DefaultGlobalConstantCategory[] = "Behavior Context: Global Constants";
+    static constexpr char DefaultGlobalMethodCategory[] = "Behavior Context: Global Methods";
+    static constexpr char DefaultClassMethodCategory[] = "Behavior Context: Class Methods";
+    static constexpr char DefaultEbusHandlerCategory[] = "Behavior Context: EBus Handlers";
+    static constexpr char DefaultEbusEventCategory[] = "Behavior Context: EBus Events";
+
     // Various Helper Methods
```

Comment:
I would remove "Behavior Context:" from these, it's too low-level for most of the target users

Reply:
done